TSPL.docc/LanguageGuide/ControlFlow.md

```diff
@@ -844,6 +932,41 @@ this `switch` statement uses a `default` case
 to match all characters other than `a` and `z`.
 This provision ensures that the `switch` statement is exhaustive.
 
+Like `if` statements,
+`switch` statements also have an expression form:
+
+```swift
+let anotherCharacter: Character = "a"
+let message = switch anotherCharacter {
+case "a":
+    "The first letter of the alphabet"
+case "z":
+    "The last letter of the alphabet"
+default:
+    "Some other character"
+}
```

Comment:
Personally I think aligning the `case` statements with the `let` looks a bit odd, I would rather it be written as:

```swift
let message = switch anotherCharacter {
    case "a":
        "The first letter of the alphabet"
    case "z":
        "The last letter of the alphabet"
    default:
        "Some other character"
}
```

or:

```swift
let message = 
    switch anotherCharacter {
    case "a":
        "The first letter of the alphabet"
    case "z":
        "The last letter of the alphabet"
    default:
        "Some other character"
    }
```

Any thoughts @airspeedswift?

Reply:
The style TSPL uses is (also used by Apple documentation and sample code projects) is that indentation follows scope.  So we don't indent `case` statements, because they don't begin a new scope — note that this is different from how many C family languages are indented.)

Changed this in 279a9ce0.